vendor-bin/php-codesniffer/composer.lock

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
Could you remove the lock files from the vendor-bin directory? This is to align with pattern used in other repositories.